protocol/alterconfigs/alterconfigs_test.go

```diff
@@ -0,0 +1,73 @@
+package alterconfigs_test
+
+import (
+	"testing"
+
+	"github.com/segmentio/kafka-go/protocol/alterconfigs"
+	"github.com/segmentio/kafka-go/protocol/prototest"
+)
+
+const (
+	v0 = 0
+	v1 = 1
+)
+
+func TestAlterConfigsRequest(t *testing.T) {
+	prototest.TestRequest(t, v0, &alterconfigs.Request{
+		ValidateOnly: true,
+		Resources: []alterconfigs.RequestResources{
+			{
+				ResourceType: 0,
```

Comment:
How about using non-zero values for integer fields? If there was a bug in the encoding or decoding logic it would likely result in the integers having their value, which we can't differentiate if the test expects zero as well.

Reply:
@achille-roussel Yes good point :+1: I agree with your idea here.

:heavy_check_mark: https://github.com/segmentio/kafka-go/pull/695/commits/ac63e80980f8fa525d9f92eb3655d1897e19c569